CorsixTH/Lua/humanoid_actions/die.lua

```diff
@@ -178,7 +178,7 @@ local action_die_tick_reaper; action_die_tick_reaper = permanent"action_die_tick
     grim_reaper:setNextAction(IdleSpawnAction(1660, point_dir):setCount(40))
 
     --Initialise the grim reaper:
-    grim_reaper:setHospital(humanoid.world:getLocalPlayerHospital())
+    grim_reaper:setHospital(humanoid.hospital)
```

Comment:
You're now relying here on the humanoid having a hospital set (it should). Makes sense.